ext/opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -97,6 +114,12 @@ def _get_metric_descriptor(
                 logger.warning(
                     "Label value %s is not a string, bool or integer", value
                 )
+
+        if self.unique_identifier:
+            descriptor["labels"].append(
+                LabelDescriptor(key="opentelemetry_uuid", value_type="STRING")
```

Comment:
Nit: "uuid" is misleading here, maybe "pid".

Reply:
I'll just use id. I think "pid" is misleading and might confuse some users.